.travis.yml

```diff
@@ -28,10 +28,18 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: CUSTOM_TEST_SCRIPT=.swift-test-parallel
```

Comment:
This is good for proving the tests pass, but I think we could replace the serial tests with the parallel ones (otherwise it's an awful lot of builds for Travis).

Reply:
Sure, I will reduce some of the serial tests to parallel ones to make sure we effectively don't add any more CI runs.